rolling/distribution.yaml

```diff
@@ -385,8 +385,8 @@ repositories:
     release:
       tags:
         release: release/rolling/{package}/{version}
-      url: https://github.com/ros2-gbp/backward_ros-release.git
-      version: 1.0.1-2
+      url: https://github.com/pal-gbp/backward_ros-release.git
```

Comment:
@saikishor IIRC our Rolling distribution infrastructure requires release repos to be under the `ros2-gbp` organization. CC @nuclearsandwich.

Reply:
@saikishor the current access is described by the configuration here: https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/pal_robotics.tf 

You can use the update release team issue template when [creating an issue](https://github.com/ros2-gbp/ros2-gbp-github-org/issues/new/choose) on that project to request changes to the team composition.

Reply:
@nuclearsandwich Thank you for the info, will request for access in an issue.

Thank you

Reply:
Requested for access : https://github.com/ros2-gbp/ros2-gbp-github-org/issues/35

Once we are given access, I will do the release again and update this PR with new changes. Thank you.

Reply:
@hidmic and @nuclearsandwich, I have made the release in the `ros2-gbp`  organization repository.